src/main/java/app/UserConfiguration.java

```diff
@@ -0,0 +1,42 @@
+package app;
+
+import java.util.List;
+
+public class UserConfiguration
+{
+    private final int port;
+    private final String stompConnectionEndpoint;
```

Comment:
It is not change request, but more explanation request. What is the difference between <code>stompConnectionEndpoint</code> from this class and <code>triggerEndpoint</code> from <code>TriggerEndpointConfiguration</code>?

Reply:
stomp connection endpoint is URL where you go to connect to stomp. As for client stomp works like this: you connect to stomp web socket. You tell him what endpoints you want to listen to. Then `stompConnectionEndpoint` is url where stomp resides and waits for you to connect. Then trigger endpoints are enpoints which will trigger/pass/cause notifications to selected `sendToEndpoints`. Did I make it any clearer or not really xD